tests/Validator/RangeTest.php

```diff
@@ -39,10 +39,12 @@ public function testIsValid()
         $this->assertEquals($this->range->isValid(1), true);
         $this->assertEquals($this->range->isValid(4), true);
         $this->assertEquals($this->range->isValid(5), true);
-        $this->assertEquals($this->range->isValid('5'), true);
+        $this->assertEquals($this->range->isValid('5'), false);
         $this->assertEquals($this->range->isValid(6), false);
         $this->assertEquals($this->range->isValid(-1), false);
         $this->assertEquals($this->range->getMin(), 0);
         $this->assertEquals($this->range->getMax(), 5);
+        $this->assertEquals($this->range->isArray(), false);
+        $this->assertEquals($this->range->getType(), 'integer');
```

Comment:
Can we use the constants from the class here instead of a hardcoded string? 🤔

Reply:
👍🏻